pkg/repository/backend_postgres.go

```diff
@@ -885,14 +884,35 @@ func (c *PostgresBackendRepository) CreateDeployment(ctx context.Context, worksp
 	return &deployment, nil
 }
 
+func (c *PostgresBackendRepository) listStubsQueryBuilder(filters types.StubFilter) squirrel.SelectBuilder {
+	qb := squirrel.StatementBuilder.PlaceholderFormat(squirrel.Dollar).Select(
+		"s.*",
+		"w.external_id AS \"workspace.external_id\"", "w.name AS \"workspace.name\"",
+	).From("stub s").
+		Join("workspace w ON s.workspace_id = w.id")
+
+	// Apply filters
+	if filters.WorkspaceID != "" {
```

Comment:
do we still need this?

Reply:
it's good to include. I need to update the http endpoint for stubs later to include workspace and workspace authentication